src/ImageProcessorCore/Formats/Tiff/TiffExifExtractor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace ImageProcessorCore.Formats
+{
+    /// <summary>
+    ///
+    /// </summary>
+    internal class TiffExifExtractor : ITiffVisitor
+    {
+        private readonly Dictionary<ushort, TiffTag> _exifTags; 
+
+        public List<ImageProperty> Properties { get; private set; }
+
+        public TiffExifExtractor()
+        {
+            Properties = new List<ImageProperty>();
+            
+            _exifTags = TiffTagRegistry.Instance.Tags
+                .Where( i => i.TagGroup == "Exif")
+                .ToDictionary(i => i.TagId);
+
+        }
+
+        public void Visit(TiffProperty property)
+        {
+            if (_exifTags.ContainsKey(property.Tag.TagId))
+            {
+                // until the image property can hold more than just a string value....
+                ImageProperty imageProperty = new ImageProperty(property.Tag.Name, property.Value.ToString());
```

Comment:
Also on a side note. The ImageProperty list is on the concrete Image class. The decoders only have the abstract ImageBase class. Are you OK with moving them to the abstract class. I'm not sure that all formats will have properties?